addons/website_slides_survey/controllers/slides.py

```diff
@@ -153,7 +153,7 @@ def _prepare_ranks_badges_values(self, **kwargs):
 
         # 3. Remove certification badge from badges and keep only certification badge linked to opened survey
         badges = values['badges'] - certification_badges
-        certification_badges = certification_badges.filtered(lambda b: b.survey_id.state == 'open')
+        certification_badges = certification_badges.filtered(lambda b: b.survey_id.active is True)
```

Comment:
Actually can you test locally if this line is at all necessary?

Archived records should be automatically filtered out by the ORM, but as there are some compute methods involved it would be best to double check.

(We will also have to adapt the above comment).

Side-note: thank you for updating the commit message, looking much better, don't forget to update it again if we end up removing this line after all.